app/controllers/hyrax/admin/admin_sets_controller.rb

```diff
@@ -2,9 +2,13 @@ module Hyrax
   class Admin::AdminSetsController < ApplicationController
     include Hyrax::CollectionsControllerBehavior
 
+    before_action :authenticate_user!
+    load_and_authorize_resource
     before_action :ensure_manager!, except: [:show]
     before_action :ensure_viewer!, only: [:show]
-    load_and_authorize_resource
+
+    # Catch permission errors
+    rescue_from Hydra::AccessDenied, CanCan::AccessDenied, with: :deny_adminset_access
```

Comment:
This plus the `deny_adminset_access` method provide a forwarding behavior more consistent with other controllers.